### DISCUSSION TITLE
Earliest version that uses `vim.uv` instead of `vim.loop`?

### DISCUSSION THREAD

**Discussion by adigitoleo** [neovim/neovim#30143] [Q&A] [answered]
I'm aware that the `vim.loop` namespace is deprecated in version 0.10(.1), but what is the earliest version where `vim.uv` is a suitable, complete replacement for `vim.loop`? I struggled to find a particular PR or commit for that.

**Comment by zeertzjq** [comment, marked as answer]
It's #22846
